src/fileformats/ocd_georef_fields.cpp

```diff
@@ -0,0 +1,461 @@
+    { OcdGrid::BRAZIL, 20, 31981 }, // SIRGAS 2000 / UTM zone 21S
+    { OcdGrid::BRAZIL, 21, 31982 }, // SIRGAS 2000 / UTM zone 22S
+    { OcdGrid::BRAZIL, 22, 31983 }, // SIRGAS 2000 / UTM zone 23S
+    { OcdGrid::BRAZIL, 23, 31984 }, // SIRGAS 2000 / UTM zone 24S
+    { OcdGrid::BRAZIL, 24, 31985 }, // SIRGAS 2000 / UTM zone 25S
+    { OcdGrid::BRITAIN, 0, 27700 }, // OSGB 1936
+    { OcdGrid::CZECHIA, 1, 5514 }, // S-JTSK Krovak East North
+    { OcdGrid::CZECHIA, 2, 28403 }, // Pulkovo 1942 / Gauss-Kruger zone 3
+    { OcdGrid::FINLAND, 2, 2392 }, // KKJ / Finland zone 2
+    { OcdGrid::FINLAND, 5, 3067 }, // TM35FIN, ETRS89 / ETRS-TM35FIN
+    { OcdGrid::FINLAND, 12, 3879 }, // ETRS89 / GK25FIN
+    { OcdGrid::FINLAND, 25, 3132 }, // ETRS89 / ETRS-GK25FIN
+    { OcdGrid::GERMANY, 2, 31466 }, // DHDN, Gauss-Krüger Zone 2
+    { OcdGrid::GERMANY, 3, 31467 }, // DHDN, Gauss-Krüger Zone 3
+    { OcdGrid::GERMANY, 4, 31468 }, // DHDN, Gauss-Krüger Zone 4
+    { OcdGrid::GERMANY, 5, 31469 }, // DHDN, Gauss-Krüger Zone 5
+    { OcdGrid::GOOGLE_TMS, 0, 900913 }, // Google Mercator
+    { OcdGrid::HUNGARY, 0, 23700 }, // HD72 / EOV
+    { OcdGrid::ICELAND, 4, 3057 }, // ISN93 / Lambert 1993
+    { OcdGrid::ISRAEL, 0, 2039 }, // Israel 1993 / Israeli TM Grid
+    { OcdGrid::SLOVAKIA, 1, 5514 }, // S-JTSK Krovak East North
+    { OcdGrid::SLOVENIA, 1, 3912 }, // MGI 1901 / Slovene National Grid
+    { OcdGrid::SWEDEN, 2, 3006 }, // SWEREF99 TM
+    { OcdGrid::SWEDEN, 3, 3007 }, // SWEREF99 12 00
+    { OcdGrid::SWEDEN, 4, 3008 }, // SWEREF99 13 30
+    { OcdGrid::SWEDEN, 5, 3009 }, // SWEREF99 15 00
+    { OcdGrid::SWEDEN, 6, 3010 }, // SWEREF99 16 30
+    { OcdGrid::SWEDEN, 7, 3011 }, // SWEREF99 18 00
+    { OcdGrid::SWEDEN, 8, 3012 }, // SWEREF99 14 15
+    { OcdGrid::SWEDEN, 9, 3013 }, // SWEREF99 15 45
+    { OcdGrid::SWEDEN, 10, 3014 }, // SWEREF99 17 15
+    { OcdGrid::SWEDEN, 11, 3015 }, // SWEREF99 18 45
+    { OcdGrid::SWEDEN, 12, 3016 }, // SWEREF99 20 15
+    { OcdGrid::SWEDEN, 13, 3017 }, // SWEREF99 21 45
+    { OcdGrid::SWEDEN, 14, 3018 }, // SWEREF99 23 15
+    { OcdGrid::SWITZERLAND, 1, 21781 }, // CH1903 / LV03 -- Swiss CH1903 / LV03
+    { OcdGrid::SWITZERLAND, 2, 2056 }, // CH1903+ / LV95 -- Swiss CH1903+ / LV95
+};
+
+/**
+ * MapperCrsRecord struct is used for passing CRS information around within the
+ * translation code.
+ */
+struct MapperCrsRecord
+{
+	/** CRS id, converted to a string via crs_string_map and passed to
+	 *  CRSTemplateRegistry::find() */
+	const MapperCrs crs_id;
+	/** CRS parameter, converted to a string and passed through to
+	 *  CRSTemplateRegistry::specValues() */
+	const int crs_param;
+};
+
+/**
+ * Plain translation from OCD grid/zone combination into possible Mapper
+ * schemes.
+ *
+ * @see translateMapperToOcd
+ */
+std::vector<MapperCrsRecord> translateOcdToMapper(const int ocd_grid_id,
+                                                  const int ocd_zone_id)
```

Comment:
The API is prepared to handle different ways of expressing the CRS in Mapper. But are we really going to use this in foreseeable time? And if yes, couldn't there be a separate function? My impression is that just returning `MapperCrsRecord` would be a fair simplification, here and in `applyGridAndZone`.